multisrc/src/main/java/eu/kanade/tachiyomi/multisrc/mangathemesia/MangaThemesiaGenerator.kt

```diff
@@ -58,6 +59,7 @@ class MangaThemesiaGenerator : ThemeSourceGenerator {
         SingleLang("KumaPoi", "https://kumapoi.me", "id", isNsfw = true),
         SingleLang("Komiku.com", "https://komiku.com", "id", className = "KomikuCom"),
         SingleLang("Kuma Scans (Kuma Translation)", "https://kumascans.com", "en", className = "KumaScans", overrideVersionCode = 1),
+        SingleLang("Legacy Scans", "https://legacy-scans.com", "fr"),
```

Comment:
Better to keep the old package name so users that have the existing one can update to the new one.

```suggestion
        SingleLang("Legacy Scans", "https://legacy-scans.com", "fr", packageName = "flamescansfr"),
```